tests/test_transformers.py

```diff
@@ -85,3 +94,18 @@ def test_transformers_are_subclassed_properly(transformer):
 def test_transformer_base_class():
     with pytest.raises(TypeError, match="Can't instantiate abstract class Transformer"):
         Transformer()
+
+
```

Comment:
Maybe add a test that confirms that when you just run the base normalise transformer that the sum of each vectors values is equal to one? Since that might be the main use-case?

Reply:
Since we are comparing the results with `sklearn.preprocessing.normalize` in the tests and assuming that function is working correctly, I don't think there is a need to check the sums as well.

Reply:
Fair enough.